contrib/opencensus-ext-azure/README.rst

```diff
@@ -114,11 +114,79 @@ You can enrich the logs with trace IDs and span IDs by using the `logging integr
         logger.warning('In the span')
     logger.warning('After the span')
 
+Metric
+~~~~~~
+
+The **OpenCensus Azure Monitor Metrics Exporter** allows you to export metrics to `Azure Monitor`_.
+
+Metrics Exporter Usage
+~~~~~~~~~~~~~~~~~~~~~~
+
+Metrics Exporter Import
+************************
+
+.. code:: python
+
+        from opencensus.ext.azure import metric_exporter
+        from opencensus.stats import stats as stats_module
```

Comment:
1. Remove partial example code.
2. Move from stats to metrics, `opencensus.stats` is being replaced by `opencensus.metrics`. If we have dependency in `stats`, we need to refactor it and move to `metrics`.